.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "ng serve",
```

Comment:
This is an interesting setup, does it boot the project up when you open it in vscode?

Reply:
Yeah when I run the ng serve --open command it runs through vscode.